test/src/test/java/org/corfudb/infrastructure/logreplication/SessionManagerTest.java

```diff
@@ -124,6 +131,7 @@ public void testSessionMgrTopologyChange() {
 
         SessionManager sessionManager = new SessionManager(topology, corfuRuntime, replicationManager, router,
                 msgHandler, pluginConfig);
+        sessionManager.getReplicationContext().setIsLeader(true);
```

Comment:
do we have a negative test where a non-leader node tries to create a session?

Reply:
Yes, L80 and L81 checks this scenario